pkg/util/azureclient/dns.go

```diff
@@ -0,0 +1,64 @@
+package azureclient
```

Comment:
I wouldn't add the mocked clients here if we are going to use them only in the fake rp.

Reply:
We are getting mocked clients for free soon via the SDK, see https://github.com/openshift/openshift-azure/issues/725

The more we add now, the harder it will be to move away in the future.

Reply:
dam! good dam! :D